frontend/src/lib/components/shared/BaseButton/styled-components.ts

```diff
@@ -80,6 +80,7 @@ export const StyledBaseButton = styled.button<RequiredBaseButtonProps>(
     justifyContent: "center",
     fontWeight: theme.fontWeights.normal,
     padding: `${theme.spacing.xs} ${theme.spacing.md}`,
+    minHeight: "38.4px",
```

Comment:
Understandable as a short term solution, but this is a definitely a strange value :upside_down_face: